tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -29,6 +31,12 @@ Given('the user has browsed to the trashbin page', function () {
     .navigateAndWaitTillLoaded()
 })
 
+When('the user browses to the trashbin page using the webUI', function () {
```

Comment:
```suggestion
When('the user browses to the trashbin page', function () {
```
Here the user is not browsing to the trashbin page using the webUI but instead directly through url

Reply:
Please see `the user has browsed to the favorites page using the webUI` to see what I mean by that.